support-frontend/assets/helpers/page/page.js

```diff
@@ -156,6 +167,29 @@ function init<S, A>(
       store.dispatch(setExperimentVariant(exp));
     });
 
+    // initiate fetch of existing payment methods
```

Comment:
I know that the query params will prevent us from actually making this call when we don't intend to, but this feels very checkout specific so I'm not sure that it should live in this function or file

Reply:
absolutely! DONE in https://github.com/guardian/support-frontend/pull/1673/commits/56e51278be3003019ab65aa1ca9f990d73fb0dcc